litestar/handlers/websocket_handlers/listener.py

```diff
@@ -52,12 +55,14 @@ class websocket_listener(WebsocketRouteHandler):
         "_receive_mode",
         "_send_mode",
         "_listener_context",
+        "accept_connection_handler",
     )
 
     def __init__(
         self,
         path: str | None | list[str] | None = None,
         *,
+        accept_connection_handler: Callable[[WebSocket], Coroutine[Any, Any, None]] = WebSocket.accept,
```

Comment:
How about just `accept_connection`? `accept_connection_handler` sounds a bit like it's the connection handler that's being accepted.

Reply:
`connection_accept_handler`?